bundles/org.openhab.binding.km200/src/main/java/org/openhab/binding/km200/internal/handler/KM200ThingHandler.java

```diff
@@ -156,10 +156,10 @@ Channel createChannel(ChannelTypeUID channelTypeUID, ChannelUID channelUID, Stri
         String itemType = "";
         String category = null;
         if (CoreItemFactory.NUMBER.equals(type)) {
-            itemType = "NumberType";
+            itemType = CoreItemFactory.NUMBER;
```

Comment:
Now local variable `itemType` could be dropped and `type` can be used directly in line 170 and 178.

Reply:
Yes, certainly

Reply:
Done